in_toto/runlib.py

```diff
@@ -87,7 +87,7 @@ def _apply_exclude_patterns(names, exclude_patterns):
   return names
 
 
-def record_artifacts_as_dict(artifacts):
+def record_artifacts_as_dict(artifacts, follow_links=False):
```

Comment:
Unrelated to this pull request:

> Paths are normalized for matching and storing by left stripping "./"

Is this behavior similar to https://github.com/theupdateframework/tuf/issues/639?  @JustinCappos recommended that we raise an exception if a path includes a leading "/"

Reply:
No, this is different. It is about normalizing/unifying relative path that might start with or without `./`
```python
>>> os.path.normpath("./relative/to/cwd") == "relative/to/cwd"
True
```
IIRC, your case is about normalizing/unifying absolute and relative paths, when the absolute root is also the relative root of a path, right?